github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```
@@ -784,7 +784,7 @@ periodics:
     testgrid-dashboards: kubevirt-periodics
     testgrid-days-of-results: "60"
   cluster: prow-workloads
-  cron: 0 40 22,10 * * *
+  cron: 0 22, 10 * * *
```

Will it work with that white space after the comma?
```suggestion
  cron: 0 22,10 * * *
```